src/App.tsx

```diff
@@ -10,9 +11,18 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/people">
+              <Route index element={<PeoplePage />} />
+              <Route path=":slug" element={<PeoplePage />} />
+            </Route>
```

Comment:
The nested `Route` structure for `/people` is correct, but ensure that the `PeoplePage` component is correctly imported and used. It seems to be imported on a non-commentable line, which might indicate an issue with its usage or import.